doc/getting-started.md

```diff
@@ -1,7 +1,5 @@
 # Getting started with Istio
 
-## This document is <span style="color:red">UNDER CONSTRUCTION</span>
-
 ## Before you begin
 
 This tutorial assumes you have a working Kubernetes of at least version 1.5.2.  Do `kubectl version` to verify
```

Comment:
My bad that I didn't review this properly in the first pass. Lets not use first person narrative in our Github docs (maintaining same style everywhere). Please switch to third person narrative.